src/components/ButtonGroup.js

```diff
@@ -44,6 +45,7 @@ const ButtonGroup = React.createClass({
             <Button
               icon={button.icon}
               key={i}
+              onClick={button.onClick}
```

Comment:
`button` is an object in the `buttons` array. I'm mapping through the array on line 39.